system/systemtest.mk

```diff
@@ -64,6 +64,11 @@ ifeq (,$(findstring $(JDK_IMPL),hotspot))
   JAVA_ARGS += -Xdump:system:events=user
 endif
 
+APPLICATION_OPTIONS :=
+ifdef JVM_OPTIONS
+  APPLICATION_OPTIONS += -jvmArgs $(JVM_OPTIONS)
```

Comment:
- instead of `+=`, we just need `=`
- Please add `$(Q)` around `$(JVM_OPTIONS)`. `$(JVM_OPTIONS)` may contain multiple jvm options.